worker/src/main/resources/application.properties

```diff
@@ -13,6 +13,18 @@ spring.jpa.properties.hibernate.temp.use_jdbc_metadata_defaults=false
 ## ------------------------------------------------------------------------------------  SPRING INTEGRATION CONFIG
 spring.integration.jdbc.initialize-schema=always
 
+## ------------------------------------------------------------------------------------  DELETE AUDIT FILES CONFIG
+# Run every hour http://www.quartz-scheduler.org/documentation/quartz-2.3.0/tutorials/crontrigger.html
+audit-files-cron=0 0 * * * ?
```

Comment:
Let's move these to `application.audit.properties` within the `audit` module and load them up via `@PropertySource`. See `QuartzSetup.java` for an example.